test/user.test.js

```diff
@@ -138,7 +138,7 @@ describe('User', function() {
       });
     });
 
-    it('Email is required', function(done) {
+    it('Email is required (email case-sensitivity on)', function(done) {
       User.create({password: '123'}, function(err) {
         assert(err);
```

Comment:
```js
if (err) return done(err);
```

I know this is not exactly related to your changes, so feel free to ignore this comment if you prefer to.